examples/arkode/F2003_serial/ark_kpr_mri_f2003.f90

```diff
@@ -571,7 +571,7 @@ program main
   type(c_ptr) :: arkode_mem                ! ARKODE memory structure
   type(c_ptr) :: inner_arkode_mem          ! ARKODE memory structure
   type(c_ptr) :: inner_stepper             ! inner stepper
-  type(c_ptr) :: BTF, BTs                  ! fast/slow method Butcher table
+  type(c_ptr) :: BTf, BTs                  ! fast/slow method Butcher table
```

Comment:
Does this PR depend on #200 (or vice-versa)?  If so, should we merge one before reviewing the other?

Reply:
This PR depends on #200 to fix the memory leaks so the Jenkins tests will pass. Merging #200 would make reviewing this PR easier.